.gitignore

```diff
@@ -2,11 +2,20 @@
 # Files already tracked by Git are not affected.
 # See: https://git-scm.com/docs/gitignore
 
+## macOS ##
+.DS_Store
+
+## Project Specific ##
+pe_reports_logging.log
+src/pe_source/data/dnstwist_output.txt
+
+
 ## Python ##
 __pycache__
 .coverage
 .mypy_cache
 .pytest_cache
 .python-version
+.vscode
```

Comment:
`.vscode` is not a Python-related file.  It should either be in a section called "IDE" or "VS Code".

Reply:
Addressed in [f58ca92](https://github.com/cisagov/pe-source/pull/1/commits/f58ca923388e9cc50052a26ab644dc043e8e595c)